ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
@leandro-lucarella-sociomantic I have reduced it to essentially a loop over all report files that concatenates them into single uploaded file and then does few curl queries - one against codecov API to get a token and later to Amazon S3 to actually upload report.

With some trial and error we could probably reproduce it but it seems somewhat fragile to me with all hard-coded urls. What do you think about simply storing the whole script inside ocean repo? It doesn't download / execute any other scripts internally AFAICS so that would work as sort of approved pinning.